boards/arm/nucleo_l452re/doc/index.rst

```diff
@@ -6,11 +6,17 @@ ST Nucleo L452RE
 Overview
 ********
 
-The Nucleo L452RE board features an ARM Cortex-M4 based STM32L452RE MCU
-with a wide range of connectivity support and configurations. Here are
-some highlights of the Nucleo L452RE board:
+The board supports a wide range of connectivity options and configurations. There
+are two variants:
 
-- STM32 microcontroller in UFQFPN32 package
+- ST Nucleo L452RE
+- ST Nucleo L452RE-P
+
+On one hand, the Nucleo L452RE board features an ARM Cortex-M4 based STM32L452RE MCU.
+On the other hand, the Nucleo L452RE-P board features an ARM Cortex-M4 based
+STM32L452RET6PU MCU. Here some highlights of these boards:
```

Comment:
I would not go into detail regarding the package of the -P version here. Both MCUs are STM32L452RE.

```suggestion
Here some highlights of the boards:
```